cmd/kanikoExecute.go

```diff
@@ -42,6 +42,12 @@ func kanikoExecute(config kanikoExecuteOptions, telemetryData *telemetry.CustomD
 }
 
 func runKanikoExecute(config *kanikoExecuteOptions, telemetryData *telemetry.CustomData, commonPipelineEnvironment *kanikoExecuteCommonPipelineEnvironment, execRunner command.ExecRunner, httpClient piperhttp.Sender, fileUtils piperutils.FileUtils) error {
+	binfmtSupported, _ := docker.IsBinfmtMiscSupportedByHost(fileUtils)
```

Comment:
for now this will always run in the Kaniko container. Do we know the result there?

Reply:
Yes we do, It checks if the hosts kernel has support for it.